scripts/convert_gs1_xml_to_json_in_dir.js

```diff
@@ -1,12 +1,14 @@
 // This script is used to convert GDSN data from Equadis in XML format
 // to a corresponding JSON structure
 
-const xml2json = require('xml2json')
-const fs = require("fs")
+const process = require('process')
```

Comment:
needs a ";"

linting does not pass, see https://github.com/openfoodfacts/openfoodfacts-server/runs/5869772770?check_suite_focus=true#step:5:39

Reply:
fixed